source/sessions/tests/README.md

```diff
@@ -238,6 +238,20 @@ and configure a `MongoClient` with default options.
 - Attempt to send a write command to the server (e.g., `insertOne`) with the explicit session passed in
 - Assert that a client-side error is generated indicating that sessions are not supported
 
+### 20. Drivers do not gossip `$clusterTime` on SDAM commands.
```

Comment:
I'm unsure about this test.  Previously we didn't have a test that asserts that cluster time _is_ gossiped by SDAM commands.  And now we have a test that it is _not_.  When we fixed this issue in the java driver we didn't end up adding an integration test for it, because it was fixed by design: in the Java driver in order to gossip cluster time you need access to the client-wide instance of `ClusterClock`.  In the fix we just removed the `ClusterClock` from the server monitor, so there in way it could unintentionally gossip.

That said, for other drivers I guess this test might be a useful one, so if you want to leave it I'm ok with that.

Reply:
I'd prefer to keep the test. It does provide some value and it at least saves implementors' time trying to answer "how should I test this? Can I even test it?".